tests/unit/legacy/api/xmlrpc/test_xmlrpc.py

```diff
@@ -862,7 +862,7 @@ def test_changelog(db_request, with_ids):
     if with_ids is not None:
         extra_args.append(with_ids)
 
-    assert xmlrpc.changelog(db_request, since - 1, *extra_args) == expected
+    assert xmlrpc.changelog(db_request, since, *extra_args) == expected
```

Comment:
This is the only change I'm not 100% sure about, but tried my best to investigate as to why it existed, and how the behavior changed with this change https://github.com/pypa/warehouse/pull/10429/files#diff-21b777d758a9add540b8c4f1c6c1e00ae3797ab426bc42e7c70cbe549036bb70L153-R159

This commit has more details [`93a002a` (#10429)](https://github.com/pypa/warehouse/pull/10429/commits/93a002a567eef94f9a8a9cb3b3bfd70fc2315d4a)